### PR TITLE
Gulp and npm

### PR DESCRIPTION
**Pull request by AndreasBrommund**
Fix build and package manager using gulp and npm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
-http://www.chartjs.org/
+## First time setup
+* Install nginx
+* Install npm 
+* Run **npm install**
+* Run **npm run setup**
+* Set nginx root path = ”(*path to iTime-web*)/www”
+* Start nginx **sudo nginx**
+
+## Folowing build
```
Comment (dskeppstedt) [inline README.md]: Typo: Should be Following builds

File: dev/js/load_index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+window.onload = function() {
+
+	startApp();
+	var data = {
+    labels: [
+        "Red",
+        "Blue",
+        "Yellow"
+    ],
+    datasets: [
```
Comment (DaRavenox) [inline dev/js/load_index.js]: Move this to another file, seems weird to have it hardcoded into load_index.js

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  " * Copyright 2017 Nick Downie\n" +
+  " * https://github.com/chartjs/Chart.js/blob/master/LICENSE.md\n" +
+  " */\n";
+
+
+//Build chart.js
+gulp.task('chart', function() {
+
+	var bundled = browserify('./node_modules/chart.js/src/chart.js', { standalone: 'Chart' })
+	.plugin(collapse)
```
Comment (DaRavenox) [inline gulpfile.js]: Formatting please

File: gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    .bundle()
+    .pipe(source('Chart.bundle.js'))
+    .pipe(insert.prepend(header))
+    .pipe(streamify(replace('{{ version }}', package.version)))
+    .pipe(gulp.dest(outDir+'js/'))
+    .pipe(streamify(uglify()))
+    .pipe(insert.prepend(header))
+    .pipe(streamify(replace('{{ version }}', package.version)))
+	.pipe(streamify(concat('Chart.bundle.min.js')))
+	.pipe(gulp.dest(outDir+'js/'));
```
Comment (DaRavenox) [inline gulpfile.js]: Formatting please

File: dev/js/load_index.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+window.onload = function() {
+
+	startApp();
+	var data = {
+    labels: [
```
Comment (dskeppstedt) [inline dev/js/load_index.js]: Why oh, why! The indentation is very broken.

Comment (dskeppstedt) [reply dev/js/load_index.js]: Please fix!

File: gulpfile.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+  return merge(bundled, nonBundled);
+
+});
+
+
+//Build html
+gulp.task('html', function() {
+
+	return gulp.src(dev+'/**/*.html')
```
Comment (DaRavenox) [inline gulpfile.js]: Too many tabs?

File: gulpfile.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+      .pipe(gulp.dest(outDir));
+
+});
+
+//Build css
+gulp.task('css', function() {
+
+	return gulp.src(dev+'style/**/*.css')
```
Comment (DaRavenox) [inline gulpfile.js]: Too many tabs?

File: gulpfile.js [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+  " *\n" +
+  " * Copyright 2017 Nick Downie\n" +
+  " * https://github.com/chartjs/Chart.js/blob/master/LICENSE.md\n" +
+  " */\n";
+
+
+//Build chart.js
+gulp.task('chart', function() {
+
+	var bundled = browserify('./node_modules/chart.js/src/chart.js', { standalone: 'Chart' })
```
Comment (dskeppstedt) [inline gulpfile.js]: Fix indentation!

File: gulpfile.js [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+  return gulp.src(dev+'js/**/*.js')
+    .pipe(concat('main.js'))
+    .pipe(gulp.dest(outDir+'js'))
+    .pipe(rename({suffix: '.min'}))
+    .pipe(gulp.dest(outDir+'js'));
+});
+
+
+
+//gulp.task('watch', function() {
```
Comment (dskeppstedt) [inline gulpfile.js:100]: Why comments?

File: package.json [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "setup": "gulp build",
+    "build": "gulp",
+	"clear": "gulp clear"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (dskeppstedt) [inline package.json]: ISC?

**Review by dskeppstedt — CHANGES_REQUESTED** [review]
Cool! Some things needs to be fixed.

**Review by dskeppstedt — APPROVED** [review]

**Pull request merged**